webpack/scenes/ContentViews/Details/Versions/Delete/RemoveSteps/CVReassignActivationKeysForm.js

```diff
@@ -102,8 +108,21 @@ const CVReassignActivationKeysForm = () => {
     cvSelectOptions,
   });
 
+  const multiCVRemovalInfo = __('The current version will be removed from the multi-CV activation keys. The reassignment will not be added to these activation keys in this step. See hammer activation-key --help for more details.');
```

Comment:
```suggestion
  const multiCVRemovalInfo = __('This environment is used in one or more multi-environment activation keys. The environment will simply be removed from the multi-environment keys. The content view and lifecycle environment you select here will only apply to single-environment activation keys. See hammer activation-key --help for more details.');
```

Reply:
Ack... I have incorporated the suggested changes and pushed the updated code.